plugin/trino-hive/src/test/java/io/trino/plugin/hive/benchmark/BenchmarkHiveFileFormat.java

```diff
@@ -461,7 +469,24 @@ public TestData createTestData(FileFormat format, double nullChance)
             }
         };
 
-        public abstract TestData createTestData(FileFormat format, double nullChance);
+        public abstract TestData createTestData(FileFormat format, double nullChance, long dataLength);
+    }
+
+    static {
+        // Pollute the JVM profile
+        try {
+            for (DataSet dataSet : DataSet.values()) {
+                for (BenchmarkFileFormat fileFormat : BenchmarkFileFormat.values()) {
+                    BenchmarkHiveFileFormat benchmark = new BenchmarkHiveFileFormat(dataSet, NONE, fileFormat);
+                    benchmark.setup(10_000); // Smaller data set allows to run more param combinations
+                    benchmark.read(new CompressionCounter());
+                    benchmark.write(new CompressionCounter());
+                }
+            }
+        }
+        catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
```

Comment:
I'm not sure this is a good way to pollute the JVM profile. Also, it makes it impossible to run a benchmark *without* mixed profiles. There's a built-in way in JMH to do so (called bulk warmup mode).

Reply:
It is done this way in BenchmarkColumnReaders in ORC.

Reply:
Did some research. The bulk warmup runs the warmup for all the possible combinations. Since there are 264 possible combinations in this benchmark every single run will take more than an hour to start making this benchmark last somewhere aroud half a month.
The proposed solution runs only a subset of combinations for a very limited time adding few seconds to every iteration.

Reply:
@martint ping